.github/workflows/native-oci-generative-ai.yml

```
@@ -0,0 +1,31 @@
+name: native-oci-generative-ai
+on:
+  push:
+    paths:
+      - 'native-oci-generative-ai/**'
+      - '.github/workflows/native-oci-generative-ai.yml'
+  pull_request:
+    paths:
+      - 'native-oci-generative-ai/**'
+      - '.github/workflows/native-oci-generative-ai.yml'
+  schedule:
+    - cron: "0 0 1 * *" # run every month
+  workflow_dispatch:
+permissions:
+  contents: read
+jobs:
+  run:
+    name: Run 'native-oci-generative-ai'
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v4
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          java-version: '21.0.3'
+          distribution: 'graalvm'
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Run 'native-oci-generative-ai'
+        run: |
+          cd native-oci-generative-ai
+          mvn clean package -Pnative
```

Use `-B`/`--batch-mode` or at least `--no-transfer-progress` for shorter build logs ([problematic example](https://github.com/graalvm/graalvm-demos/actions/runs/9061198825/job/24892496069?pr=264#step:4:76)):
```suggestion
          mvn --batch-mode clean package -Pnative
```